build-tools/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks/GenerateWixFile.cs

```diff
@@ -108,9 +108,22 @@ public override bool Execute ()
 				packWriter.WriteStartElement ("Directory");
 				packWriter.WriteAttributeString ("Id", "packs");
 				packWriter.WriteAttributeString ("Name", "packs");
+				packWriter.WriteAttributeString ("FileSource", packs_dir);
```

Comment:
Would it be possible to include a "snippet" of what the new XML output looks like with these changes?

Reply:
This line adds the full path such as:
```
<Directory Id="packs" Name="packs" FileSource="C:\Users\myuser\android-toolchain\dotnet\packs">
```
It was not explicitly needed before because `RecurseDirectory()` adds sub-directories with `FileSource`, but I added it because the rest of the elements specify `FileSource`.

The second block looks like:
```
<Directory Id="templatepacks" Name="template-packs" FileSource="C:\Users\myuser\android-toolchain\dotnet\template-packs">
  <Component Id="SC983E605827BDDA589CA4DFF082E29CDFEC2CD059B4AF3C01421CC95568A8D74">
    <File Id="SC983E605827BDDA589CA4DFF082E29CDFEC2CD059B4AF3C01421CC95568A8D74" Name="Microsoft.Android.Templates.11.0.100-ci.dotnet-new.259.nupkg" KeyPath="yes" />
  </Component>
</Directory>
```
MSI has weird rules around the `Id`, which is why I used hashes for files.